### PR TITLE
Rename everything 'resin' to 'balena'

### LINKED ISSUE #42: Rename everything

### PR DESCRIPTION
**Pull request by pimterry**
This still depended on the old SDK. This PR fixes that, and updates everything to balena throughout.

No breaking functional changes, the only functional one is that due to updates in the SDK settings libraries, this moves from using `~/.resin/cache` as the image cache to using `~/.balena/cache` instead.

The tests currently fail, because they try to test against the prod API, and the certificate is still invalid. They should start immediately working as soon as that's resolved.

Fixes #42 
Change-type: minor
Signed-off-by: Tim Perry <tim@balena.io>

### CODE REVIEW

File: tests/cache.spec.coffee [hunk 1]
```diff
@@ -20,27 +20,27 @@ describe 'Cache:', ->
...
 
 			describe 'given valid slugs', ->
 
 				beforeEach ->
-					@getManifestBySlugStub = m.sinon.stub(resin.models.device, 'getManifestBySlug')
+					@getManifestBySlugStub = m.sinon.stub(balena.models.device, 'getManifestBySlug')
 					@getManifestBySlugStub.withArgs('raspberry-pi').returns Promise.resolve
 						yocto:
-							fstype: 'resin-sdcard'
+							fstype: 'balena-sdcard'
```
Comment (shaunmulligan) [inline tests/cache.spec.coffee]: Not sure this is gonna change yet, should check with the OS team

Comment (pimterry) [reply tests/cache.spec.coffee]: True - reverted.

It doesn't actually matter here, the code just tests `!= 'zip`, but more accurate test data is definitely better anyway.

**Comment by pdcastro** [comment]
@pimterry, the required ResinCI/npm/store-npm check is failing:
```
npm ERR! notarget No matching version found for balena-sdk@^11.0.0
```

**Comment by pimterry** [comment]
@pdcastro yep, that's expected - we can't merge & release SDK v11 until the new API is live and usable (hopefully later today). At that point, this PR should suddenly start building successfully.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+balena-image-manager
 -------------------
 
-[![npm version](https://badge.fury.io/js/resin-image-manager.svg)](http://badge.fury.io/js/resin-image-manager)
-[![dependencies](https://david-dm.org/resin-io/resin-image-manager.png)](https://david-dm.org/resin-io/resin-image-manager.png)
-[![Build Status](https://travis-ci.org/resin-io/resin-image-manager.svg?branch=master)](https://travis-ci.org/resin-io/resin-image-manager)
-[![Build status](https://ci.appveyor.com/api/projects/status/2nxg1uydksvey0g8?svg=true)](https://ci.appveyor.com/project/jviotti/resin-image-manager)
+[![npm version](https://badge.fury.io/js/balena-image-manager.svg)](http://badge.fury.io/js/balena-image-manager)
+[![dependencies](https://david-dm.org/balena-io/balena-image-manager.png)](https://david-dm.org/balena-io/balena-image-manager.png)
+[![Build Status](https://travis-ci.org/balena-io/balena-image-manager.svg?branch=master)](https://travis-ci.org/balena-io/balena-image-manager)
```
Comment (pdcastro) [inline README.md]: Have the svg and png files actually been renamed over there?

Comment (pimterry) [reply README.md]: As in https://github.com/resin-io-modules/resin-sync/pull/157#pullrequestreview-168267862 - these are all autogenerated from github data, so once things get renamed here they'll magically start working.

Comment (pdcastro) [reply README.md]: 👍

**Pull request merged**